static/app/views/starfish/views/webServiceView/endpointList.tsx

```diff
@@ -109,6 +113,30 @@ class EndpointList extends Component<Props, State> {
       );
     }
 
+    if (field === TIME_SPENT_IN_SERVICE) {
+      const cumulativeTime = Number(dataRow['sum(transaction.duration)']);
+      const cumulativeTimePercentage = Number(dataRow[TIME_SPENT_IN_SERVICE]);
+      return (
+        <Tooltip
+          title={t(
+            'This endpoint accounts for %s of the cumulative load on your web service',
```

Comment:
It's a little weird that "cumulative time" is explained as "cumulative load", I think! Which one is it?

Reply:
I like time!